ci/travis/build-docker-images.sh

```diff
@@ -16,6 +17,45 @@ docker_push() {
         echo "Skipping docker push because it's in PR environment."
     fi
 }
+build_and_push_tags() {
+    # $1 image-name (Dockerfile directory)
+    # $2 tag
+    for GPU in "" "-gpu" 
+    do 
+        BASE_IMAGE=$(if [ "$GPU" ]; then echo "nvidia/cuda:11.0-cudnn8-runtime-ubuntu18.04"; else echo "ubuntu:focal"; fi;)
+        SPECIFIC_NAME="rayproject/$1:$2$GPU"
```

Comment:
It took me some time to grasp what it means. How about `FULL_NAME_WITH_TAG` and `LATEST_...`

Reply:
Sounds good!